stock.go

```diff
@@ -79,6 +79,27 @@ func (s *Stock) Shutdown() {
 	s.close <- 1
 }
 
+// sendMessage responds to users who @ the bot
+func sendMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
+
+    // ignores itself
+    if m.Author.ID == s.State.User.ID {
+        return
+    }
+
+    // listens for mention
+    for _, user := range m.Mentions {
+        if user.ID == s.State.User.ID {
+            s.ChannelMessageSend(m.ChannelID, "Bot  online;") 
+        }
+    }
+        
+    // the xxxxx should be what the bot responds to, with the 'response here' being the response
+    // if m.Content == "xxxxx" {
+    //    s.ChannelMessageSend(m.ChannelID, "Response here*")
+    //}
```

Comment:
Can we remove this completely? Not a big fan of commented out code. 

```suggestion
```

Reply:
Yea thats fine, might be worth having something about it somewhere in the readme though?